lib/deprecation-cop-view.coffee

```diff
@@ -90,10 +81,10 @@ class DeprecationCopView extends ScrollView
 
   serialize: ->
     deserializer: @constructor.name
-    uri: @getUri()
+    uri: @getURI()
 
   handleGrimUpdated: =>
-    @refreshCallsButton.show()
+    @updateCalls()
```

Comment:
Just to be on the safe side, perhaps this should be wrapped in a `_.debounce` call here with maybe a 1 second timeout so that any deprecations occurring while the view is open don't lock up Atom as the view is re-rendered.

I could see this happening perhaps via a deprecation path happening on a mouse event and Atom could seize up as the deprecation is continuously hit and the view rebuilt synchronously.

http://underscorejs.org/#debounce

What do you think?